website/docs/r/is_vpc.html.markdown

```diff
@@ -53,6 +71,39 @@ Review the argument references that you can specify for your resource.
 - `default_network_acl_name` - (Optional, String) Enter the name of the default network access control list (ACL).
 - `default_security_group_name` - (Optional, String) Enter the name of the default security group.
 - `default_routing_table_name` - (Optional, String) Enter the name of the default routing table.
+- `dns` - (Optional, List) The DNS configuration for this VPC.
+  
+  Nested scheme for `dns`:
```

Comment:
Note - This feature is a Select Location availability invitation only feature. If used in other regions may lead to inconsistencies in state management.